packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -91,7 +91,7 @@
     "jest-junit": "^13.0.0",
     "jest-react-hooks-shallow": "^1.4.1",
     "jest-svg-transformer": "^1.0.0",
-    "mendix": "^9.13.43286",
+    "mendix": "^9.16.49269",
```

Comment:
@tamelbrun, should we use `9.17` here?

Reply:
It's not strictly necessary, as the client typings for this feature are already available in 9.16. This is also mentioned in the PR description, by the way. (The 9.17 typings won't be available until after the release of Studio Pro 9.17.0, I didn't want to wait for that if it wasn't necessary.)

Reply:
But I'll doublecheck if there are any changes in the client typings between 9.16 and 9.17, just to be sure.

Reply:
For this specific PR, which only uses the widget property types, we don't need to bump to 9.17. But the _release_ of the PW tools should definitely bump the client types to 9.17, as there are additional changes in the client typings that need to be included. But if that is not already standard practice for a minor release of the PW tools, it definitely should be :).